packages/parser/lib/css_lexer.re

```diff
@@ -352,44 +350,61 @@ let unreachable = () =>
 
 let (~:) = f => f();
 
-module Tokenizer = {
-  let consume_whitespace = buf =>
-    switch%sedlex (buf) {
-    | Star(whitespace) => Parser.WS
-    | _ => Parser.WS
-    };
+let (let.ok) = Result.bind;
 
-  let lexeme = Sedlexing.Utf8.lexeme;
+let consume_whitespace = buf =>
+  switch%sedlex (buf) {
+  | Star(whitespace) => Parser.WS
+  | _ => Parser.WS
+  };
 
-  // https://drafts.csswg.org/css-syntax-3/#consume-an-escaped-code-point
-  let consume_escaped = buf => {
-    switch%sedlex (buf) {
-    // TODO: spec typo? No more than 5?
-    | Rep(hex_digit, 1 .. 6) =>
-      let hex_string = "0x" ++ lexeme(buf);
-      let char_code = int_of_string(hex_string);
-      let char = uchar_of_int(char_code);
-      let _ = consume_whitespace(buf);
-      char_code == 0 || is_surrogate(char_code)
-        ? Error((Uchar.rep, Tokens.Invalid_code_point)) : Ok(char);
-    | eof => Error((Uchar.rep, Tokens.Eof))
-    | any => Ok(lexeme(buf))
-    | _ => ~:unreachable
-    };
+let lexeme = Sedlexing.Utf8.lexeme;
+
+// https://drafts.csswg.org/css-syntax-3/#consume-an-escaped-code-point
+let consume_escaped = buf => {
+  switch%sedlex (buf) {
+  // TODO: spec typo? No more than 5?
+  | Rep(hex_digit, 1 .. 6) =>
+    let hex_string = "0x" ++ lexeme(buf);
+    let char_code = int_of_string(hex_string);
+    let char = uchar_of_int(char_code);
+    let _ = consume_whitespace(buf);
+    char_code == 0 || is_surrogate(char_code)
+      // U+FFFD is a character used as a substitute for an uninterpretable character from another encoding
```

Comment:
Where this U+FFFD is being treated?

Reply:
It is referenced as a string on the next two lines. Though I am not so convinced with it.